tracing-opentelemetry/README.md

```diff
@@ -0,0 +1,70 @@
+# Tracing OpenTelemetry
```

Comment:
This is a nit, and I'm open to changing it later so as to not block this PR, but most of the other crates' READMEs have the following headers:

- Overview
- Usage
  - (sometimes examples)
- License
- Contributing

It's fine (and encouraged!) to have other sections as well, but it would be nice to have the common ones consistently.

Reply:
Can do